sos/plugins/nis.py

```diff
@@ -22,7 +22,9 @@ class Nis(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
     plugin_name = 'nis'
     profiles = ('identity', 'services')
 
-    files = ('/var/yp',)
+    files = ('/var/yp', '/etc/ypserv.conf')
+
+    packages = 'ypserv'
```

Comment:
The `packages` member must be an iterable type...

I never bothered to make this change since every NIS pacakge I've ever seen has owned `/var/yp`. Since other distros could, potentially, not do that I don't really have an objection to it but the patch needs to be respun to fix the type.